<?xml version="1.0" ?>
<launch>    
    <param name="gazebo"                   value="false"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_op2"/>
    
    <param name="offset_file_path"         value="$(find op2_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find op2_manager)/config/OP2.robot"/>
    <param name="init_file_path"           value="$(find op2_manager)/config/dxl_init.yaml"/>
    <param name="device_name"              value="/dev/ttyUSB0"/>
    
    <param name="action_file_path"         value="$(find op2_manager)/config/motion_4096.bin" />
    <param name="init_pose_file_path"      value="$(find op2_manager)/config/ini_pose.yaml" />

    <param name="/robotis/direct_control/check_collision" value="false" />
    
    <!-- OP2 Manager -->
    <node pkg="op2_manager" type="op2_manager" name="op2_manager" output="screen">
	    <param name="angle_unit" value="30" />
	  </node>
</launch>
